Gems/ScriptCanvas/Code/Include/ScriptCanvas/Translation/GraphToLuaUtility.cpp

```diff
@@ -23,6 +25,23 @@
 
 namespace GraphToLuaUtilityCpp
 {
+    class ScopedLocale
```

Comment:
Asking @o3de/sig-core-maintainers if there's a good engine-wide place for this, seems like other areas might run into this issue.